demo/znkQuestionReport/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>znk Question Report</title>
+    <!-- bower:css -->
+    <link rel="stylesheet" href="Swiper/dist/css/swiper.css" />
+    <link rel="stylesheet" href="angular-material/angular-material.css" />
+    <!-- endbower-->
+
+    <link rel="stylesheet" href="main.css">
+
+</head>
+<body ng-app="demo" layout="column"  ng-controller="Main as vm">
+<znk-question-report report-data="vm.questionData" style="margin: 5em;"></znk-question-report>
```

Comment:
we usually don't use inline style
maybe it is ok here for the demo....but generally not...